packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -25,7 +25,18 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
           [`.${IsFocusVisibleClassName} &:focus`]: {
             // Can't use getFocusStyle because it doesn't support wrapping links
             // https://github.com/OfficeDev/office-ui-fabric-react/issues/4883#issuecomment-406743543
-            outline: `1px solid ${theme.palette.neutralSecondary}`
+            // A box-shadow allows the focus rect to wrap links that span multiple lines
+            // and helps the focus rect avoid getting clipped.
+            boxShadow: `0 0 0 1px ${theme.palette.neutralSecondary} inset`
```

Comment:
I believe this should have `outline: none`

Reply:
There's already an outline: 'none' right above this on line 21 that's applied for all cases... is it necessary to have it restated in the focus selector?

Reply:
Hmm. I think so?

If you inspect a focused link, does it have "outline: none" in the rule? 

I am mostly concerned with specificity. It probably would be a more correct thing to move the outline within the focus styling. Outline none doesn't really impact non-focused scenarios for default styling.

Reply:
When I keep outline:none outside of the focus styling and inspect a focused link, there is "outline: none" in the rule: 
![image](https://user-images.githubusercontent.com/4161791/52093883-af0e6880-2571-11e9-9be1-2a0e34882c58.png)
But when I move it inside the focus styling, it's not in the rule...
![image](https://user-images.githubusercontent.com/4161791/52093850-9736e480-2571-11e9-8238-3beb9b6dc231.png)

Reply:
For now, I've kept inside the focus styling but let me know what you think about these screenshots - I feel like outline:none was followed better when it was outside...